Sources/BrowserServicesKit/Autofill/AutofillUserScript+SourceProvider.swift

```diff
@@ -103,7 +103,7 @@ public class DefaultAutofillSourceProvider: AutofillUserScriptSourceProvider {
 
     private func buildReplacementsData() -> ProviderData? {
         guard let filteredPrivacyConfigData = filteredDataFrom(configData: privacyConfigurationManager.currentConfig,
-                                                               keepingTopLevelKeys: ["features", "unprotectedTemporary"],
+                                                               keepingTopLevelKeys: ["features", "unprotectedTemporary", "language"],
```

Comment:
`language` isn't actually a property of the privacy config, so there is no key to keep here to provide to `contentScope` 🙂   

It is however already available as a property in `userPreferences`: 
```
{
  "language": "en",
  "globalPrivacyControlValue": true,
  "sessionKey": "FCFAA3D3-6F11-4307-BBC2-B308935DC389",
  "features": {
    "autofill": {
      "settings": {
        "featureToggles": {
          "emailProtection_incontext_signup": true,
          "inlineIcon_credentials": true,
          "inputType_creditCards": true,
          "credentials_saving": true,
          "third_party_credentials_provider": true,
          "emailProtection": true,
          "inputType_identities": true,
          "password_generation": true,
          "inputType_credentials": true
        }
      }
    }
  },
  "debug": false,
  "platform": {
    "name": "macos"
  }
}
```

Reply:
🤔 Iiiiinteresting! Without adding `language` I swear I didn't get the language on the JS side. My memory is likely failing me. I'll check it out!